lix/packages/sdk/src/mock/mock-csv-plugin.ts

```diff
@@ -31,27 +31,13 @@ export const mockCsvPlugin: LixPlugin = {
 				// update the cell
 				(parsed.data[rowIndex] as any)[columnIndex] = text;
 			} else {
-				if (change.parent_id === undefined) {
-					throw new Error(
-						"Expected a previous change to exist if a value is undefined (a deletion)",
-					);
-				}
-				// TODO possibility to avoid querying the parent change?
-				const parent = await lix.db
-					.selectFrom("change")
-					.innerJoin("snapshot", "snapshot.id", "change.snapshot_id")
-					.selectAll("change")
-					.select("snapshot.value")
-					.where("change.id", "=", change.parent_id)
-					.executeTakeFirstOrThrow();
-
-				const { rowIndex, columnIndex } = parent.value as unknown as Cell;
-				(parsed.data as any)[rowIndex][columnIndex] = "";
-				// if the row is empty after deleting the cell, remove it
```

Comment:
Querying the snapshot is not needed anymore if the entity_id exists on the change itself 😎